src/dotnet/APIView/APIViewWeb/Account/SamplesRevisionOwnerRequirementHandler.cs

```diff
@@ -1,20 +1,21 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
 using System.Threading.Tasks;
+using APIViewWeb.LeanModels;
 using Microsoft.AspNetCore.Authorization;
 
 namespace APIViewWeb
 {
-    public class UsageSampleOwnerRequirementHandler : IAuthorizationHandler
+    public class SamplesRevisionOwnerRequirementHandler : IAuthorizationHandler
```

Comment:
Can we simplify this design is to use a parent class  `RevisionModel` which is inherited by both `SamplesRevisionModel` and `ReviewRevisionModel` so that only one class is required to check revision owner reqiurement?